apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -59,9 +60,12 @@ class OrderBlocksAnswerData(TypedDict):
     depends: list[str]  # only used with DAG grader
     group_info: GroupInfo  # only used with DAG grader
     distractor_bin: NotRequired[str]
+    distractor_feedback: Optional[str]
+    is_correct: bool
     uuid: str
 
 
+FIRST_WRONG_TYPES = [FeedbackType.FIRST_WRONG, FeedbackType.FIRST_WRONG_VERBOSE]
```

Comment:
```suggestion
FIRST_WRONG_TYPES = frozenset((FeedbackType.FIRST_WRONG, FeedbackType.FIRST_WRONG_VERBOSE))
```
Not completely sure this will work but nice to have an immutable type rather than a list.

Reply:
ya, it works. Done.